app/models/ingest_job.rb

```diff
@@ -537,14 +537,23 @@ def read_parse_logfile(filepath, delete_on_read: true, range: nil)
   #   - (Hash) => Hash of job statistics to use with IngestJob#log_to_mixpanel
   def get_job_analytics
     file_type = self.study_file.file_type
+
+    if self.study_file.remote_location.present?
+      trigger = 'sync'
+    else
+      trigger = 'upload'
+    end
```

Comment:
declaring variables inside an if block that later get used outside the 'if' block can make code tougher to read as finding the source of a variable is more difficult (especially depending on code folding in the IDE).  It also risks harder-to-catch typos if one of the  declarations inside the if branch is misspelled.  Avoiding it will keep our codebase more generally legible
```suggestion
    trigger = self.study_file.remote_location.present? ?  'sync' : 'upload'
```

Reply:
Concise, too -- I like it.